### PR TITLE
Added check to PV generator component to check for the same surfaces

### PR DESCRIPTION
**Pull request by antonszilasi**

### CODE REVIEW

File: src/Honeybee_Honeybee.py [hunk 1]
```diff
@@ -47,7 +47,11 @@
 
 ghenv.Component.Name = "Honeybee_Honeybee"
 ghenv.Component.NickName = 'Honeybee'
+<<<<<<< HEAD
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Honeybee.py]: There is a conflict line here.

File: src/Honeybee_Honeybee.py [hunk 2]
```diff
@@ -47,7 +47,11 @@
...
+ghenv.Component.Message = 'VER 0.0.58\nNOV_18_2015'
+=======
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Honeybee.py]: the same here

File: src/Honeybee_Honeybee.py [hunk 3]
```diff
@@ -47,7 +47,11 @@
...
 ghenv.Component.Message = 'VER 0.0.58\nNOV_19_2015'
+>>>>>>> fabf9d2e8b577b4831cd1915a302d193725356cf
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Honeybee.py]: and here!

**Comment by mostaphaRoudsari** [comment]
@antonszilasi there is a couple of conflict lines in Honeybee_Honeybee. Did you even change anything in honeybee_honeybee?

**Comment by chriswmackey** [comment]
@antonszilasi,
This has probably resulted due to some changes I made to HB_HB yesterday:
https://github.com/mostaphaRoudsari/Honeybee/commit/fabf9d2e8b577b4831cd1915a302d193725356cf
Can you sync and resubmit one more time?
-Chris

**Pull request closed without merging**